lib/spack/llnl/util/lang.py

```diff
@@ -1072,3 +1072,15 @@ def __exit__(self, exc_type, exc_value, tb):
         # Suppress any exception from being re-raised:
         # https://docs.python.org/3/reference/datamodel.html#object.__exit__.
         return True
+
+
+class ClassProperty(object):
```

Comment:
can we call this `classproperty` to be consistent?

Reply:
Fixed in 8c0a82e